### ISSUE TITLE
iPad keyboard fn keys

### ISSUE THREAD

**Issue by ulkuderner** [ish-app/ish#1634]
I know there is meta-option key changer for external keyboard but since option key is also used by some other functionalities it would be great to use fn keys with cmd key like blink app.

**Comment by reinhart1010** [comment]
Yeah, I also need this to run [mitzasql](https://github.com/vladbalmos/mitzasql) MySQL TUI client on iSH.
